apps/condo/domains/user/utils/serverSchema/requestLimitHelpers.js

```diff
@@ -29,7 +33,28 @@ const checkDailyRequestLimitCountersByPhone = async (context, prefix, phone) =>
     }
 }
 
+const checkDailyRequestLimitCountersByUser = async (context, prefix, userId) => {
+    const key = [prefix, 'userId', userId].filter(Boolean).join(':')
```

Comment:
Can we move prefix for this counter into consts? If we need to add it to ResetUserLimitAction, it will be helpful (for some reason I think that sometimes QA will ask to reset this counter)

Reply:
added reset of findOrganizationsByTin in ResetUserLimitAction